package.json

```diff
@@ -560,6 +560,11 @@
                     "id": "vscode-edge-devtools-view.targets",
                     "name": "Targets"
                 },
+                {
+                    "id": "vscode-edge-devtools-view.mirror-editing",
+                    "name": "CSS Mirror Editing",
+                    "when": "config.vscode-edge-devtools.browserFlavor == Canary"

```

Comment:
We probably shouldn't base this off of the `browserFlavor` since it would require us to go back and update it once it hits Dev/Beta/Stable, etc.  This actually reminded me that I needed a browser compatibility check for the vscode commands I registered, so I created a `isCdnHostedTools` context [here](https://github.com/microsoft/vscode-edge-devtools/pull/462/files#diff-95909d339267cba030f94b894f49de695bc6e910f1262da13d851ee866d5109cR460).  You can probably just rebase and use that here as well.

Reply:
+1 on adding the label as it makes it more obvious
-1 on pre-populated questions, as people tend to delete most template content

This is excellent!